R/pokehelperr.R

```diff
@@ -1,3 +1,6 @@
+library(glue)
+library(dplyr)
+library(stringr)
```

Comment:
When making a package, we shouldn't import packages in this manner. We need to add them to the package's dependencies via use_package(<package_name>).

Reply:
See https://r-pkgs.org/whole-game.html#use_package.

Reply:
Oh sorry, forgot to remove them. Thanks